### PR TITLE
[QNN] Add qnn op for abs to fix wrong scale on quantize

### PR DESCRIPTION
**Pull request by mikepapadim**
Prior implementation caused the FQI pass to produce the following graph when `abs` was used:

```
  %14 = qnn.dequantize(%13, 6.4712f /* ty=float32 */, 119 /* ty=int32 */, axis=1) /* ty=Tensor[(1, 960, 512, 64), float32] */;
  %15 = abs(%14) /* ty=Tensor[(1, 960, 512, 64), float32] */;
  %16 = qnn.quantize(%15, 3.46196f /* ty=float32 */, 0 /* ty=int32 */, out_dtype="uint8", axis=1) /* ty=Tensor[(1, 960, 512, 64), uint8] */;
```
It seems that the output scale and zero point are different from the input scale and zero point. So, we doubled the precision. 

By adding a qnn op for `abs` we manage requantize on the right values.

```
 def @main(%x: Tensor[(1, 960, 512, 64), int8] /* ty=Tensor[(1, 960, 512, 64), int8] */) -> Tensor[(1, 960, 512, 64), int8] {
  %0 = qnn.abs(%x, 6.4712f /* ty=float32 */, 119 /* ty=int32 */, 6.4712f /* ty=float32 */, 0 /* ty=int32 */) /* ty=Tensor[(1, 960, 512, 64), int8] */;
  qnn.requantize(%0, 6.4712f /* ty=float32 */, 119 /* ty=int32 */, 3.46196f /* ty=float32 */, 0 /* ty=int32 */, out_dtype="int8") /* ty=Tensor[(1, 960, 512, 64), int8] */
}
```

@sfvaroglu @mbrookhart @AndrewZhaoLuo @anwang2009

### CODE REVIEW

File: tests/python/relay/test_pass_fake_quantization_to_integer.py [hunk 1]
```diff
@@ -27,6 +27,9 @@ def compare_fq_to_int(expr, args, allow_rounding_error=False):
     mod = tvm.IRModule.from_expr(expr)
     mod = tvm.relay.transform.InferType()(mod)
     mod_int = tvm.relay.transform.FakeQuantizationToInteger()(mod)
+    print(mod)
+    print("-----")
+    print(mod_int)
```
Comment (mbrookhart) [inline tests/python/relay/test_pass_fake_quantization_to_integer.py:32]: Remove extra prints

**Review by mbrookhart — APPROVED** [review]
Left some debug prints, remove those. Otherwise, LGTM

File: tests/python/relay/test_pass_fake_quantization_to_integer.py (hunk 1 above)
Comment (mikepapadim) [reply tests/python/relay/test_pass_fake_quantization_to_integer.py:32]: oups. I am cleaning this now

**Comment by mikepapadim** [comment]
@tvm-bot rerun

**Comment by mbrookhart** [comment]
Thanks @mikepapadim

**Pull request merged**